Postman/Postman.js

```diff
@@ -1,54 +1,49 @@
-"use strict";
+const Q = require('q');
+const _ = require('underscore');
+const moment = require('moment-timezone');
 
-var Q = require('q');
-var _ = require('underscore');
-var moment = require('moment-timezone');
-
-var MailToUser = require('Postman/lib/MailToUser');
-var Librarian = require('Librarian/Librarian');
-var log = require('common/log');
-var PERIODICAL_DAY = require('common/constant').PERIODICAL_DAY;
+const MailToUser = require('Postman/lib/MailToUser');
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).